components/AddLostItem.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "styled-components";
+import { useState } from "react";
+import { lostItems } from "./Db";
+import { nanoid } from "nanoid";
+import Image from "next/image";
+
+export default function AddLostItem() {
+  const [description, setDescription] = useState("");
+  const [location, setLocation] = useState("");
+  const [name, setName] = useState("");
+  const [lost, setLost] = useState(lostItems);
+  const [img, setImage] = useState("/assets/MrBean.png");
+
+  const randomId = nanoid();
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    const newLostItem = { name, description, location, img, randomId };
```

Comment:
const newLostItem = { name, description, location, img, randomId } -> you dont need this anymore